tests/FakeItEasy.Tests/Core/CallOrderingContextTests.cs

```diff
@@ -0,0 +1,146 @@
+﻿namespace FakeItEasy.Tests.Core

```

Comment:
Actually, I had intentionally put the interface in the root namespace. My thinking was that the interface might be used explicitly by the user; some people just don't like `var`, so they would do this:

    using (ICallOrderingContext context = A.CallOrderingContext())

The class is internal, so it probably belongs in `Core`.